### PR TITLE
To fix lint and formatting issues in the guardio-insurance-sample-application

### PR DESCRIPTION
**Pull request by Achintha444**

### CODE REVIEW

File: b2b-sample/guardio-insurance-sample-application/.eslintrc.json [hunk 1]
```diff
@@ -1,3 +1,22 @@
...
+        "max": 1,
+        "maxEOF": 1
+      }
+    ],
+    "eol-last": [
+      "error",
+      "always"
+    ]
+  }
+}
```
Comment (thivi) [inline b2b-sample/guardio-insurance-sample-application/.eslintrc.json:22]: Add a new line.

File: b2b-sample/guardio-insurance-sample-application/styles/globals.css [hunk 2]
```diff
@@ -30,5 +31,6 @@ a {
 
 * {
     box-sizing: border-box;
-    font-family: "Nunito Sans", Menlo, Monaco, Lucida Console, Liberation Mono, DejaVu Sans Mono, Bitstream Vera Sans Mono, Courier New, monospace;
-}
+    font-family: "Nunito Sans", Menlo, Monaco, Lucida Console, Liberation Mono, DejaVu Sans Mono,
+        Bitstream Vera Sans Mono, Courier New, monospace;
+}
```
Comment (thivi) [inline b2b-sample/guardio-insurance-sample-application/styles/globals.css:36]: Add a new empty line

File: b2b-sample/guardio-insurance-sample-application/util/util/apiUtil/localResErrors.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+function notPostError(res) {
+    return error404(res, 'Cannot request data directyly.');
+}
+
+function dataNotRecievedError(res) {
+    return error404(res, 'Error occured when requesting data.');
+}
+
+module.exports = { notPostError, dataNotRecievedError }
```
Comment (thivi) [inline b2b-sample/guardio-insurance-sample-application/util/util/apiUtil/localResErrors.js:31]: Add a new empty line

**Review by thivi — APPROVED** [review]

File: b2b-sample/guardio-insurance-sample-application/components/settingsComponents/homeComponet/homeComponent.js [hunk 4]
```diff
@@ -49,18 +50,16 @@ export default function HomeComponent(props) {
                 </div>
             </Panel>
             {
-                me == null ?
-                    <Panel bordered>
+                me == null
+                    ? <Panel bordered>
                         <div>Add the user attributes in created the application to display user details</div>
                     </Panel>
-                    :
```
Comment (lashinijay) [inline b2b-sample/guardio-insurance-sample-application/components/settingsComponents/homeComponet/homeComponent.js:56]: `Add the user attributes in the created application to display user details`

File: b2b-sample/guardio-insurance-sample-application/pages/404.js [hunk 5]
```diff
@@ -40,11 +39,12 @@ export default function Custom404() {
 
             <div className={style.errorMainDiv}>
 
-              <Image src={errorImage} width={600} alt="404 image"/>
+              <Image src={errorImage} width={600} alt="404 image" />
 
               <p style={{
                 textAlign: 'center'
-              }}><b>The page your searching seems to be missing.</b> <br />You can go back, or contact our <a>Customer Service</a> team if you need any help</p>
+              }}><b>The page your searching seems to be missing.</b> <br />
```
Comment (lashinijay) [inline b2b-sample/guardio-insurance-sample-application/pages/404.js:46]: `The page you are searching seems to be missing`

File: b2b-sample/guardio-insurance-sample-application/pages/404.js [hunk 6]
```diff
@@ -40,11 +39,12 @@ export default function Custom404() {
...
+                You can go back, or contact our <a>Customer Service</a> team if you need any help</p>
```
Comment (sandushi) [inline b2b-sample/guardio-insurance-sample-application/pages/404.js:47]: Add a `.` at the end of the line

File: b2b-sample/guardio-insurance-sample-application/util/util/apiUtil/localResErrors.js [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+function error404(res, msg) {
+    return res.status(404).json(msg);
+}
+
+function notPostError(res) {
+    return error404(res, 'Cannot request data directyly.');
```
Comment (sandushi) [inline b2b-sample/guardio-insurance-sample-application/util/util/apiUtil/localResErrors.js:24]: check the spellings here ?  `directly` ?

File: b2b-sample/guardio-insurance-sample-application/pages/404.js (hunk 5 above)
Comment (Achintha444) [reply b2b-sample/guardio-insurance-sample-application/pages/404.js:46]: Fixed with https://github.com/wso2/samples-is/pull/416/commits/5ffcaa54b0986e1d52996996832e3832ed3d6c9a

File: b2b-sample/guardio-insurance-sample-application/pages/404.js (hunk 6 above)
Comment (Achintha444) [reply b2b-sample/guardio-insurance-sample-application/pages/404.js:47]: Fixed with https://github.com/wso2/samples-is/pull/416/commits/5ffcaa54b0986e1d52996996832e3832ed3d6c9a

File: b2b-sample/guardio-insurance-sample-application/util/util/apiUtil/localResErrors.js (hunk 7 above)
Comment (Achintha444) [reply b2b-sample/guardio-insurance-sample-application/util/util/apiUtil/localResErrors.js:24]: Fixed with https://github.com/wso2/samples-is/pull/416/commits/00fcf2af0c800b9bfc234e95534cb746916ccd07

File: b2b-sample/guardio-insurance-sample-application/components/settingsComponents/homeComponet/homeComponent.js (hunk 4 above)
Comment (Achintha444) [reply b2b-sample/guardio-insurance-sample-application/components/settingsComponents/homeComponet/homeComponent.js:56]: Fixed with https://github.com/wso2/samples-is/pull/416/commits/00fcf2af0c800b9bfc234e95534cb746916ccd07

**Pull request merged**